pkg/server/etcd/watch.go

```diff
@@ -212,18 +212,18 @@ func (w *watcher) List(ctx context.Context, id int64, r *etcdserverpb.WatchCreat
 	klog.InfoS("RANGE STREAM", "watcher", w.id, "watch", id, "key", r.Key, "end", r.RangeEnd, "rev", r.StartRevision)
 	ch, err := w.backend.ListByStream(ctx, r.Key, r.RangeEnd, uint64(r.StartRevision*-1))
 	if err != nil {
-		klog.ErrorS(err, "list by stream failed", "watcher", w.id, "watch", id, "key", r.Key, "end", r.RangeEnd, "rev", int64(r.StartRevision*-1))
+		klog.ErrorS(err, "list by stream failed", "watcher", w.id, "watch", id, "key", r.Key, "end", r.RangeEnd, "rev", r.StartRevision*-1)
 		w.metricCli.EmitCounter("watch.backend.list_stream.err", 1)
 		w.Cancel(id, err, true)
 		return
 	}
 
 	for watchResponse := range ch {
-		revision := int64(r.StartRevision * -1)
+		revision := r.StartRevision * -1
```

Comment:
many refactor and style changes due to lint, maybe we could enhance the ci workflow of integration tests.

Reply:
because golangci-lint provides lint checks which is more stricter, it exposes some problems that we don't aware before.